metadata-json-lint.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'metadata-json-lint'
-  s.version     = '0.0.18'
+  s.version     = '0.0.19'
```

Comment:
once this works, should we release a version `1.0.0`? i'm fairly certain this gem is stable enough to warrant that.